src/Kafka/ManageKafkaPermissions/components/ReviewPermissionsTable.tsx

```diff
@@ -0,0 +1,129 @@
+import { VFC } from "react";
+import { useTranslation } from "react-i18next";
+
+import {
+  TableHeader,
+  Table,
+  TableBody,
+  TableVariant,
+  ICell,
+  cellWidth,
+} from "@patternfly/react-table";
+import { Label, Button, Tooltip } from "@patternfly/react-core";
+
+import TrashIcon from "@patternfly/react-icons/dist/js/icons/trash-icon";
+import { EnhancedAclBinding } from "../types";
+import { resourceCell, permissionOperationCell, CellBuilder } from "./Cells";
+
+export type ReviewPermissionsTableProps = {
+  acls: RemovableEnhancedAclBinding[];
+  onChangeAcls: (acls: RemovableEnhancedAclBinding[] | []) => void;
+  selectedAccountId?: string;
+  onRemoveAcl: (acl: RemovableEnhancedAclBinding) => void;
+};
+
+export type RemovableEnhancedAclBinding = EnhancedAclBinding & {
+  removed: boolean;
+  index: number;
+};
+
+export const ReviewPermissionsTable: VFC<ReviewPermissionsTableProps> = ({
+  selectedAccountId,
+  acls = [],
+  onRemoveAcl,
+}) => {
+  const { t } = useTranslation(["manage-kafka-permissions"]);
+
+  if (selectedAccountId === undefined || acls.length === 0) {
+    return <></>;
+  }
+
+  const principalCell: CellBuilder<RemovableEnhancedAclBinding> = (acl) => {
+    const renderRemoveButton = () => (
+      <Tooltip
+        content={t(
+          "manage_permissions_dialog.assign_permissions.remove_row_help"
+        )}
+      >
+        <Button
+          variant="link"
+          icon={<TrashIcon />}
+          onClick={() => onRemoveAcl(acl)}
+        />
+      </Tooltip>
+    );
```

Comment:
@HemaHG has a PR open for this exact component: https://github.com/redhat-developer/app-services-ui-components/pull/164

Should we merge her PR and then use that component here instead?

Reply:
yes I create the remove component to use in the permission table